api_docs/construct-narrow.md

```diff
@@ -65,12 +65,31 @@ filters did.
 
 ## Narrows that use IDs
 
+### Message IDs
+
 The `near` and `id` operators, documented in the help center, use message
 IDs for their operands.
 
 * `near:12345`: Search messages around the message with ID `12345`.
 * `id:12345`: Search for only message with ID `12345`.
 
+**Changes**: Prior to Zulip 8.0 (feature level 193), the message ID
+operand with the `id` operator needed to be encoded as a string. As of
+this feature level, the server accepts both string and integer formats
+for this operand.
```

Comment:
The "near" operator is validated as a string only, I don't know if we want to note that here.

Reply:
Yeah. Perhaps like this:

```
The message ID operand for the `id` operator may be encoded as
either a number or a string. The message ID operand for the `near` operator
must be encoded as a string.

**Changes**: Prior to Zulip 8.0 (feature level 193), the message ID
operand for the `id` operator needed to be encoded as a string.
```

which would also help keep the non-"Changes" text self-contained as a description of the current API.

Reply:
Yeah, that's much clearer. Updating as such, thanks!